src/backend/distributed/sql/udfs/citus_lock_waits/latest.sql

```diff
@@ -21,6 +21,8 @@ citus_dist_stat_activity_with_node_id AS
     citus_dist_stat_activity.distributed_query_host_port = pg_dist_node.nodeport
 )
 SELECT
+ waiting.global_pid as waiting_gpid,
+ blocking.global_pid as blocking_gpid,
  waiting.pid AS waiting_pid,
  blocking.pid AS blocking_pid,
  waiting.query AS blocked_statement,
```

Comment:
any thoughts on simplifying this view by e.g. removing hostname and port, or node ID?

Reply:
It's also not entirely clear to me whether PID refers to the process ID on the worker or the coordinator. Perhaps it doesn't matter either way?

If we could indicate where the block is happening that would be useful.

Reply:
> any thoughts on simplifying this view by e.g. removing hostname and port, or node ID?

I'll work on the definition on the view as a follow-up PR.

Currently, we still rely on dist. tx's nodeId. We should rely on the gpid's nodeId, otherwise commands that are not in a tx block may not show the node information properly.

Stay tuned :)